pkg/server/container_status.go

```diff
@@ -70,6 +74,20 @@ func toCRIContainerStatus(container containerstore.Container, spec *runtime.Imag
 			reason = errorExitReason
 		}
 	}
+	if verbose {
+		info = make(map[string]string)
+		m, _ := json.Marshal(status.Pid)
```

Comment:
Should not ignore error.

Reply:
it's verbose debug status.. you want me to fail if I can't give them all of it?

Reply:
1. Marshal should not fail, if it does, there is some error in our code logic. We should expose that kind of error.
2. Even though this is called `verbose`, it is very important for `crictl` users, so we still want to generate responsive information.

Reply:
done